test/e2e/create_test.go

```diff
@@ -108,6 +108,25 @@ func (tc *testContext) createWindowsInstanceConfigMap(machines *mapi.MachineList
 	return nil
 }
 
+// validateWindowsInstanceConfigMap validates the windows-instance ConfigMap
+func (tc *testContext) validateWindowsInstanceConfigMap(expectedCount int) error {
+	windowsInstances, err := tc.client.K8s.
+		CoreV1().
+		ConfigMaps(tc.namespace).
+		Get(context.TODO(), wiparser.InstanceConfigMap, metav1.GetOptions{})
```

Comment:
Please don't introduce such verbosity given we are doing this in other places.

Reply:
I missed a **not** in my previous comment 😄

Reply:
Addressed.